### PR TITLE
Updates for the contribution guide to reflect opening PRs

### PR DESCRIPTION
**Pull request by savannahostrowski**
I will update the aka link for the website's "contribute" button once the PR is merged.

Closes https://github.com/Azure/azure-dev/issues/2064

### CODE REVIEW

File: website/docs/1-faq/3-use-azd-template.md [hunk 1]
```diff
@@ -7,9 +7,6 @@ With `azd-templates`, the path from "code to cloud" is just 3 steps:
  - Deployment = `azd up` to deploy it
  - Customization = modify app code or config
 
-![Recommended azd workflow](https://learn.microsoft.com/azure/developer/azure-developer-cli/media/overview/workflow.png)
-
-
 :::tip READ THE DOCS
-Learn more about [about Getting Started with azd](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/get-started?tabs=bare-metal%2Cwindows&pivots=programming-language-nodejs)
+Learn more about [about getting Started with azd](https://aka.ms/azd)
 :::
```
Comment (hemarina) [inline website/docs/1-faq/3-use-azd-template.md:12]: Perhaps Learn more about [getting started with azd]?

File: .github/PULL_REQUEST_TEMPLATE/contribute-template.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+# New `azd` template submission
+Fill this out if you want your template to be added to the awesome-azd gallery!
+
+<!-- Please ensure that your PR includes the following metadata! All fields are mandatory unless explicitly marked as optional -->
+
+- [ ] Added an entry to https://github.com/Azure/awesome-azd/blob/main/website/src/data/users.tsx that includes:
+    - [ ] **Template title** - A short title that reflects the local application stack that someone could use to get their application on Azure (e.g. "Containerized React Web App with Java API and MongoDB")
+    - [ ] **Description** - 1-2 sentence description of the architecture (e.g. Azure services) or solution that is defined by the template.
+    - [ ] **Architecture Diagram or Application Screenshot** - Used as display image for gallery card-- should include all services and their connections ([example](https://github.com/Azure-Samples/todo-csharp-sql/blob/main/assets/resources.png))
```
Comment (hemarina) [inline .github/PULL_REQUEST_TEMPLATE/contribute-template.md]: Could we also remind the user to save the image in `image` folder?

File: website/yarn.lock [hunk 3]
```diff
@@ -0,0 +1,8050 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (hemarina) [inline website/yarn.lock]: Do we want to add this file?

Comment (savannahostrowski) [reply website/yarn.lock]: I don't think so! I committed by mistake!

**Pull request merged**